estate_account/models/estate_property.py

```diff
@@ -0,0 +1,27 @@
+from odoo import models, Command
+
+
+class InheritedEstateProperty(models.Model):
+    _inherit = "estate.property"
+
+    def action_sell_property(self):
+        for record in self:
+            invoice_dictionary = {
+                "partner_id": record.buyer_id.id,
+                "move_type": "out_invoice",
+                "journal_id": self.env['account.journal'].search([("code", "=", "INV")]).id,
```

Comment:
Why do you need a journal? What happen if there is none? 
You could also add `limit=1` to improve the perf